src/api/axiosInstance.ts

```
@@ -1,22 +1,19 @@
+// src/api/axiosInstance.ts
+
 import axios from 'axios';
-import type {
-  AxiosError,
-  AxiosInstance,
-  AxiosRequestConfig,
-  InternalAxiosRequestConfig,
-} from 'axios';
+import type { AxiosError, AxiosInstance, AxiosRequestConfig, InternalAxiosRequestConfig } from 'axios';
 import { STORAGE_KEYS } from '@/utils/storageKeys';
+import { ENDPOINTS } from './endpoints';
```

ESLint

[error] 'ENDPOINTS' is defined but never used.

(@typescript-eslint/no-unused-vars)